src/parser/test/ScannerTest.cpp

```diff
@@ -354,6 +354,10 @@ TEST(Scanner, Basic) {
         CHECK_SEMANTIC_VALUE("\"He\\\nllo\"", TokenType::STRING, "He\nllo"),
         CHECK_SEMANTIC_VALUE("\"\\\"Hello\\\"\"", TokenType::STRING, "\"Hello\""),
 
+        CHECK_SEMANTIC_VALUE("'Hello'", TokenType::STRING, "Hello"),
+        CHECK_SEMANTIC_VALUE("'\"Hello\"'", TokenType::STRING, "\"Hello\""),
+        CHECK_SEMANTIC_VALUE("'\\'Hello\\''", TokenType::STRING, "'Hello'"),
+
         // escape Normal character
```

Comment:
"he\'llo"-> he'llo
"he'llo" -> he'llo or syntax error?

Reply:
I added `vice versa` in the description of this PR.